frontend/components/Dashboard/Dashboard.jsx

```diff
@@ -90,6 +131,23 @@ function Graph() {
     categories: [1991, 1992, 1993, 1994, 1995, 1996, 1997, 1998, 1999],
     data: [30, 40, 45, 50, 49, 60, 70, 91],
   })
+
+
+  const context = useContext(UserAuth)
+  function fetchTotalUserLinks(){
+      return axios.get(`http://localhost:5000/minify/all/user`,{
```

Comment:
Use the env var to fetch API url

Reply:
> Use the env var to fetch API url

same with this one
![Capture](https://user-images.githubusercontent.com/82874704/167296297-a266a2c0-53c4-4f80-86b9-64eeb8dc5fa2.JPG)

Reply:
https://github.com/BRAVO68WEB/url-minify/pull/150#discussion_r867492134